sdk/tests/conformance/resources/uniformBlockShader.frag

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2015 The Chromium Authors. All rights reserved.
```

Comment:
Sorry I didn't notice this before, but please use the Khronos copyright header for these new files. You can use sdk/tests/conformance/resources/gl-object-get-calls.js as a template.